astropy/modeling/functional_models.py

```diff
@@ -786,6 +786,53 @@ def inverse(self):
         return self.__class__(slope=new_slope, intercept=new_intercept)
 
 
+class Planar2D(Fittable2DModel):
```

Comment:
Isn't this just a special case of `Polynomial2D`? So, should it really subclass `Polynomial2D` and belong in `polynomial.py`?